avocado/utils/kernel.py

```diff
@@ -72,7 +72,8 @@ def download(self):
         full_url = self.URL + self.SOURCE.format(version=self.version)
         self.asset_path = asset.Asset(full_url, asset_hash=None,
                                       algorithm=None, locations=None,
-                                      cache_dirs=self.data_dirs).fetch()
+                                      cache_dirs=[self.work_dir],
+                                      expire=None).fetch()
```

Comment:
Ouch, missed this change. Focused in expire and didn't notice `cache_dirs` was changed. Indeed `self.data_dirs` seems the correct options here.